docs/user-guide/my-time-tracking/README.md

```diff
@@ -0,0 +1,110 @@
+---
+sidebar_navigation:
+  title: My time tracking
+  priority: 950
+description: Get an overview of time logs in a calendar or list view. Easily log time directly from My time tracking page.
+keywords: time tracking, log time, time spent, calendar, time spent calendar, time logs
+---
+
+# My time tracking
+
+## Overview
+The *My time tracking* module offers a clear overview of your personal time entries. You can quickly log time directly from this page. To access it, select **My time tracking** from the left-hand menu.
+
+![My tracking page overview in OpenProject](openproject_user_guide_my_time_tracking_module_overview.png)
+
+### My time tracking page filters
+
+1. On the *My time tracking* page, you’ll find the following interface elements:
+   1. **Zen mode** button — toggles a distraction-free view.
+   2. **Tab selector** — switch between *Calendar* and *List* views.
+   3. **Zoom level dropdown menu** — choose the time scale for the view. Following options are available:
+      - Day
+      - Work Week
+      - Week
+      - Month
+   4. **Displayed date range** — shows the currently visible timeframe.
+   5. **Navigation arrows** — move to the previous or next time period.
+   6. **Today** button — jump directly to the current day.
+   7. **+ Log time** button — add a new time entry.
+
+
+
+![My time tracking module filters explained](openproject_user_guide_my_time_tracking_module_filters_explained.png)
+
+### Calendar view
+
+The *Calendar* view opens by default if start and finish times for time entries have been enabled in [System Administration](../../system-admin-guide/time-and-costs). You can adjust the zoom level using the dropdown menu, with options for **Day**, **Work week**, **Week**, or **Month**.
+
+Existing time entries appear on the corresponding days they were logged. Entry colors reflect the type of work package associated with the logged time.
+
+You can log time by selecting a time range anywhere on the calendar or by using the **+ Log time** button.
```

Comment:
I am missing some information here regarding the top section inside the calendar that allows you to log time in a day without specific start or finish dates.